### ISSUE TITLE
Implement date picker for web

### ISSUE THREAD

**Issue by flochtililoch** [Instawork/hyperview#454]
Currently, the date picker component is only supported on mobile. We should implement a working fallback for web.
See `src/core/components/date-time-picker/index.web.js` in #455.

**Comment by adamstep** [comment]
Would it make sense to use the HTML5 version? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

**Comment by flochtililoch** [comment]
yes, most likely. I did give it a try with a simple code change but the UX needs some work. I think we should probably reconsider the extra handling that adds the bottom sheet UI to wrap the date picker, and instead render the field inline, especially now that a non-spinner UI exists on iOS:

![Simulator Screen Shot - iPhone 13 mini - 2022-07-17 at 16 49 48](https://user-images.githubusercontent.com/309515/179429797-c17e7f41-fb37-401a-b826-5e1a26025f2b.png)
